codespell_lib/data/dictionary.txt

```diff
@@ -8228,6 +8232,8 @@ cursos->cursors, cursor,
 cursot->cursor
 cursro->cursor
 curvelinear->curvilinear
+cusstom->custom
+cusstoms->customs
```

Comment:
If you don't mind adding these.
`cusstomer->customer`
`cusstomers->customers`
`cusstomizable->customizable`
`cusstomization->customization`
`cusstomize->customize`
`cusstomized->customized`

  * https://github.com/search?q=cusstomer&type=code
  * https://github.com/search?q=cusstomers&type=code
  * https://github.com/search?q=cusstomizable&type=code
  * https://github.com/search?q=cusstomization&type=code
  * https://github.com/search?q=cusstomize&type=code
  * https://github.com/search?q=cusstomized&type=code